app/notification.go

```diff
@@ -274,7 +320,7 @@ func (a *App) SendNotifications(post *model.Post, team *model.Team, channel *mod
 				continue
 			}
 
-			if a.userAllowsEmail(profileMap[id], channelMemberNotifyPropsMap[id], post) {
+			if notificationsForCRT.Email.Contains(id) || a.userAllowsEmail(profileMap[id], channelMemberNotifyPropsMap[id], post) {
```

Comment:
On line 309 we add `notificationsForCRT.Email` to `emailReceipients`, and here we're looping over `emailReceipients` so this `Contains` check isn't required.

Reply:
It's because `channelMemberNotifyPropsMap` may not allow the email, but `email_threads` notify prop may allow it.
hm, I'll add the check in `a.userAllowsEmail` and remove the `Contains` from here.